pkg/ovs/openflow/interfaces.go

```diff
@@ -240,8 +249,12 @@ type FlowBuilder interface {
 	MatchCTStateInv(isSet bool) FlowBuilder
 	MatchCTStateDNAT(isSet bool) FlowBuilder
 	MatchCTStateSNAT(isSet bool) FlowBuilder
+	// Deprecated.
```

Comment:
the comment should suggest which alternative is. For example:
// Deprecated: Use MatchCTMarkData instead.

Reply:
You are correct, I have already changed the signature from structure to pointer for Range object. Then I would remove the deprecated APIs directly.